fusor-ember-cli/app/controllers/subscriptions/management-application.js

```diff
@@ -49,6 +58,12 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, {
       //POST /customer_portal/consumers?owner=#{OWNER['key']}, {"name":"#{RHCI_DISTRIBUTOR_NAME}","type":"satellite","facts":{"distributor_version":"sat-6.0","system.certificate_version":"3.2"}}
       var url = ('/customer_portal/consumers?=' + ownerKey);
 
+      if (isInvalidMgmtAppName) {
+        this.set('showWaitingMessage', false);
+        this.set('showErrorMessage', true);
+        return !isInvalidMgmtAppName;  
```

Comment:
Might want to return a promise rejection here in case calling code is expecting something